docs/contributing/style-guide.md

```diff
@@ -1,7 +1,595 @@
+
+</td>
+      <td>
+
+```java title="Custom Title"
+RemoteWebDriver driver = new RemoteWebDriver(url, capabilities);
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+## Code References
+
+The Sauce Labs Open Source Team created a plugin for use with Docusaurus to allow us to reference code on Github
+rather than duplicating code in this repo. Ideally all code displayed in the Sauce Labs documentation points to code in
+one of the `demo-<language>` repos on [Sauce Labs Training Github Org](https://github.com/saucelabs-training/?q=demo#org-repositories)
+When referencing code, Include the language, "reference" and a title indicating what the code is.
+The URL for the link can be for the entire file, or include specific line numbers at the end.
+Additionally, the referenced code should reference a tag name rather than just a branch name (which can easily break) or
+a commit hash (which is hard to version). For example, the `demo-java` repo has a tag for `website-1.0`, so the reference
+link should specify this.
+
+<table class="code">
+  <tbody>
+    <tr>
+      <td>
+
+**Markdown**
+
+</td>
+      <td>
+
+    ```java reference title="Example Test"
+    https://github.com/saucelabs-training/demo-java/blob/website-1.0/selenium-examples/src/test/java/com/saucedemo/junit5/SauceBindingsTest.java#L35-L43
+    ```
+
+</td>
+</tr>
+<tr>
+<td>
+
+**Display**
+
+</td>
+      <td>
+
+```java reference title="Example Test"
+https://github.com/saucelabs-training/demo-java/blob/website-1.0/selenium-examples/src/test/java/com/saucedemo/junit5/SauceBindingsTest.java#L35-L43
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+## Admonitions
+
+There are 4 types of [Docusaurus Admonitions](https://docusaurus.io/docs/markdown-features/admonitions):
+* Note - Relevant information.
+* Tip - A user should do this.
+* Caution - A user should pay attention to this.
+* Warning - A user might do something dangerous!
+
+<table class="code">
+  <tbody>
+    <tr>
+      <td>
+
+**Markdown**
+
+</td>
+      <td>
+
+```markdown
+:::note
+Relevant information for you.
+:::
+
+:::tip
+You should do this.
+:::
+
+:::caution
+You should probably pay attention to this.
+:::
+
+:::warning
+You are about to do something dangerous!
+:::
+```
+
+</td>
+</tr>
+<tr>
+<td>
+
+**Display**
+
+</td>
+      <td>
+
+:::note
+Relevant information for you.
+:::
+
+:::tip
+You should do this.
+:::
+
+:::caution
+You should probably pay attention to this.
+:::
+
+:::warning
+You are about to do something dangerous!
+:::
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+## Badges
+
+Badges are color-coded images that alert a reader to special considerations for a given section. 
+We currently support 2 types of badges:
+
+GOLD indicates that a page is deprecated or out-of-date in some way, for example:
+
+* DEPRECATED
+
+BLUE indicates that the feature being documented is limited in its scope or availability, for example:
+
+* ENTERPRISE ONLY
+* IOS ONLY
+* LIVE TESTING ONLY
+* BETA
+* EARLY ACCESS
+
```

Comment:
I have never managed to add this to our internal guidelines, and our badges have been a bit of an evolving entity for some time, but often, when the badge (such as `REAL DEVICES ONLY` is in a table cell (such as a table of property definitions), we set it in the <small></small> tag so it is not so overwhelming.You can see an example of what I mean here: https://docs.saucelabs.com/testrunner-toolkit/configuration/cypress#tunnel 9Look at the `tunnel` property.

Reply:
I split the badges and highlights into H3's with examples